<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Notification Manager Configuration</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>62</y>
     <width>610</width>
     <height>499</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="0" >
     <widget class="QGroupBox" name="geometry_box" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Notification Window Geometry</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="0" column="2" >
        <widget class="QLineEdit" name="percent_height" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Percent width</string>
         </property>
         <property name="buddy" >
          <cstring>percent_width</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QLineEdit" name="percent_width" />
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Percent height</string>
         </property>
         <property name="buddy" >
          <cstring>percent_height</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" >
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" >
        <spacer name="verticalSpacer_8" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item rowspan="2" row="0" column="1" >
     <widget class="QGroupBox" name="position_box" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Notification Window Pop-up Position</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3" >
       <item row="0" column="0" >
        <widget class="QRadioButton" name="upper_right_radio" >
         <property name="text" >
          <string>Upper right</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QRadioButton" name="lower_right_radio" >
         <property name="text" >
          <string>Lower right</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <spacer name="verticalSpacer_3" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" >
        <widget class="QGroupBox" name="manual_position" >
         <property name="title" >
          <string>Manual</string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_4" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>X coordinate</string>
            </property>
            <property name="buddy" >
             <cstring>startx</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QLineEdit" name="startx" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_7" >
            <property name="text" >
             <string>Y coordinate</string>
            </property>
            <property name="buddy" >
             <cstring>starty</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLabel" name="label_8" >
            <property name="text" >
             <string>:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QLineEdit" name="starty" />
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QGroupBox" name="stacking_box" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Notification window stacking direction</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5" >
       <item row="0" column="0" >
        <widget class="QRadioButton" name="stacking_up" >
         <property name="text" >
          <string>Up</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QRadioButton" name="stacking_down" >
         <property name="text" >
          <string>Down</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <spacer name="verticalSpacer_4" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item rowspan="2" row="2" column="1" >
     <widget class="QGroupBox" name="animation_box" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Animation Parameters</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>Total animation time</string>
         </property>
         <property name="buddy" >
          <cstring>animation_time</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLabel" name="label_10" >
         <property name="text" >
          <string>:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QLineEdit" name="animation_time" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_11" >
         <property name="text" >
          <string>Time quanta (frame time)</string>
         </property>
         <property name="buddy" >
          <cstring>time_quanta</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLabel" name="label_12" >
         <property name="text" >
          <string>:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QLineEdit" name="time_quanta" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_13" >
         <property name="text" >
          <string>Notification window lifetime</string>
         </property>
         <property name="buddy" >
          <cstring>lifetime</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLabel" name="label_14" >
         <property name="text" >
          <string>:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <widget class="QLineEdit" name="lifetime" />
       </item>
       <item row="3" column="0" >
        <spacer name="verticalSpacer_5" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1" >
        <spacer name="verticalSpacer_6" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2" >
        <spacer name="verticalSpacer_7" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="1" >
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="save_button" >
        <property name="text" >
         <string>Save</string>
        </property>
        <property name="icon" >
         <iconset>
          <normaloff>/usr/share/pnm/icons/save.xpm</normaloff>/usr/share/pnm/icons/save.xpm</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="discard_button" >
        <property name="text" >
         <string>Quit</string>
        </property>
        <property name="icon" >
         <iconset>
          <normaloff>/usr/share/pnm/icons/exit.png</normaloff>/usr/share/pnm/icons/exit.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" >
     <widget class="QGroupBox" name="skin_box" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Skin</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_15" >
         <property name="text" >
          <string>UI file path</string>
         </property>
         <property name="buddy" >
          <cstring>skin_path</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLabel" name="label_16" >
         <property name="text" >
          <string>:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QLineEdit" name="skin_path" />
       </item>
       <item row="0" column="3" >
        <widget class="QToolButton" name="browse_skin_path" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <spacer name="verticalSpacer_9" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" >
        <spacer name="verticalSpacer_10" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2" >
        <spacer name="verticalSpacer_11" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3" >
        <spacer name="verticalSpacer_12" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QGroupBox" name="manager_box" >
      <property name="title" >
       <string>Notification Manager Options</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_8" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_17" >
         <property name="text" >
          <string>Manager lifetime</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLabel" name="label_18" >
         <property name="text" >
          <string>:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QLineEdit" name="manager_lifetime" />
       </item>
       <item row="1" column="0" >
        <spacer name="verticalSpacer_13" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" >
        <spacer name="verticalSpacer_14" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2" >
        <spacer name="verticalSpacer_15" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="open_conf_file" />
    <addaction name="save_conf_file" />
    <addaction name="separator" />
    <addaction name="quit_configurator" />
   </widget>
   <addaction name="menuFile" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>561</y>
     <width>610</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>28</y>
     <width>610</width>
     <height>34</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="open_conf_file" />
   <addaction name="save_conf_file" />
   <addaction name="separator" />
   <addaction name="quit_configurator" />
  </widget>
  <action name="save_conf_file" >
   <property name="icon" >
    <iconset>
     <normaloff>/usr/share/pnm/icons/save.xpm</normaloff>/usr/share/pnm/icons/save.xpm</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save Config File</string>
   </property>
  </action>
  <action name="quit_configurator" >
   <property name="icon" >
    <iconset>
     <normaloff>/usr/share/pnm/icons/exit.png</normaloff>/usr/share/pnm/icons/exit.png</iconset>
   </property>
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
  <action name="open_conf_file" >
   <property name="icon" >
    <iconset>
     <normaloff>/usr/share/pnm/icons/open.png</normaloff>/usr/share/pnm/icons/open.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open Config File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
